app/build.gradle

```diff
@@ -44,6 +45,9 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
 
+    //DownloadImage
+    implementation 'com.github.bumptech.glide:glide:4.11.0'
+    implementation 'com.squareup.picasso:picasso:2.5.2'
```

Comment:
Why both?

Reply:
I was comparing the work of picasso and forgot to delete, sorry. Fixed

Reply:
Cool, no sorry is needed